Libraries/LibWeb/HTML/HTMLElement.cpp

```diff
@@ -924,7 +925,20 @@ TokenizedFeature::NoOpener HTMLElement::get_an_elements_noopener(StringView targ
     if (!link_types.contains_slow("opener"sv) && Infra::is_ascii_case_insensitive_match(target, "_blank"sv))
         return TokenizedFeature::NoOpener::Yes;
 
-    // 3. Return false.
+    // 3. If url's scheme is "blob":
+    if (url.scheme() == "blob"sv) {
```

Comment:
This looks suspiciously like it could be a spec bug where it is not checking that blob_url_entry is not null here, see similar bug which was reported then fixed here: https://github.com/whatwg/html/commit/54828a7de260c105091bef852aa6283e66e642cd

Reply:
i.e this line becomes checking if blob url entry is not null instead

Reply:
i made spec PR https://github.com/whatwg/html/pull/10962

Reply:
Thanks Shannon! I'll update it with your fix.